client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPlugin.js

```diff
@@ -610,7 +610,7 @@ function CloudLink(props) {
     return null;
   }
 
-  const query = `?process=${processId}&version=all&active=true&incidents=true`;
+  const query = `?process=${processId}&version=all&active=true&incidents=true&utm_source=modeler&utm_medium=referral`;
```

Comment:
I don't know if Google Analytics will run on Operate.
Even if the params won't break anything, we should not provide them here until we have to, because UTM params in productive work environments may reduce trust

Reply:
:+1:

Reply:
I was about to suggest that we automate this in the backend but it seems it's not an easy fix.

Reply:
Makes sense. Will remove